### PR TITLE
fix: Add RTL support to app heading.

### PR DESCRIPTION
**Pull request by falghamdi125**
### ☑️ Resolves  

-  #1850 

Hello everyone,

This PR adjusts app heading margins and adds support for RTL layouts.

### 🖼️ Screenshots

🏚️ Before | 🏡 After
| -- | -- |
| image |  image |

💡   Note: This pull request is most effective when the Nextcloud-Vue  [PR:6260]( https://github.com/nextcloud-libraries/nextcloud-vue/pull/6260)  is approved.

Best

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: src/views/ActivityAppFeed.vue [hunk 1]
```diff
@@ -222,7 +222,7 @@ watch(props, () => {
 		font-size: 20px;
 		line-height: 44px; // to align height with the app navigation toggle
 		// Align with app navigation toggle
-		margin: var(--app-navigation-padding, 8px) 0 0 calc(2 * var(--app-navigation-padding, 8px) + 44px);
+		margin-inline: calc(2 * var(--app-navigation-padding, 8px) + 44px) var(--app-navigation-padding, 8px);
```
Comment (artonge) [inline src/views/ActivityAppFeed.vue:225]: I understand that previously, `margin-top` and `margin-left` were set, and now that `margin-inline-start` `margin-inline-end` are set, which are equivalent of `margin-left` and `margin-right`. Am I right? If so, don't we need to keep the `margin-top`?

Comment (falghamdi125) [reply src/views/ActivityAppFeed.vue:225]: I am not sure about the top margin.

Keeping the top margin is ok but it is not aligned with the toggle button. 

image

Setting the top margin to zero aligns the title with the toggle button. See:
image

Probably the designers are looking for comfortable margins around app content more than alignment with external elements. It guess it is safe to maintain the top margin.

Comment (artonge) [reply src/views/ActivityAppFeed.vue]: Your second screenshot looks good indeed. Let's keep it that way :).

**Comment by falghamdi125** [comment]
Build files have been added to the commit @artonge

**Comment by artonge** [comment]
Thanks a lot @falghamdi125 !

We have a Nextcloud Talk instance where we invite Nextcloud contributors to be able to communicate more quickly. If you like, I could give you a guest account via the email address in your commits?

**Pull request merged**